lib/ui/view/login/login_viewmodel.dart

```diff
@@ -43,12 +41,16 @@ class LoginViewModel extends BaseViewModel {
     const endpoint = '/auth/login';
     if (email.text == '' || password.text == '') {
       loading(false);
+      //Hides the keyboard for the failure snackbar to be visible
+      FocusScope.of(context).unfocus();
```

Comment:
Remove flutter/material import and adjust to fit.

Reply:
changes made